client/my-sites/plans/jetpack-plans/product-lightbox/product-details.tsx

```diff
@@ -0,0 +1,46 @@
+import { useMobileBreakpoint } from '@automattic/viewport-react';
+import { useState } from 'react';
+import FoldableCard from 'calypso/components/foldable-card';
+import { SelectorProduct } from '../types';
+import DescriptionList from './description-list';
+
+type ProductDetailsProps = {
+	product: SelectorProduct;
+};
+
+const ProductDetails: React.FC< ProductDetailsProps > = ( { product } ) => {
+	const isMobile = useMobileBreakpoint();
+	const [ expandedDetailType, setExpandedDetailType ] = useState( '' );
+	const productDetails = [
+		{ type: 'includes', title: 'Includes', items: product.whatIsIncluded },
```

Comment:
I assume `Includes` here should be translated?

Reply:
Oh yes, thanks for reminding